jsk_arc2017_common/node_scripts/set_location_in_rosparam.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
```

Comment:
Ok, now I understand the necessity. +1, but please add 2 line spaces before `if __main__`, and sample into samples dir.